src/test/java/org/opensearch/security/IntegrationTests.java

```diff
@@ -1074,4 +1074,16 @@ public void testMonitorHealth() throws Exception {
         RestHelper rh = nonSslRestHelper();
         Assert.assertEquals(HttpStatus.SC_OK, rh.executeGetRequest("_cat/health", encodeBasicHeader("picard", "picard")).getStatusCode());
     }
+
+    // TODO: SOMETHING LIKE THIS
```

Comment:
I'd recommend creating the new tests inside of the the new integration test framework in a new file 'ServiceAccountTests` side by side with `OnBehalfOfJwtAuthenticationTest`, what do you think?

Reply:
Yes! This is what im doing haha. I was just leaving this comment for self-reminder

Reply:
Because some of the test setup here is not what we want.